src/main.py

```diff
@@ -37,6 +38,11 @@ def load_preferences(self):
     def connect_actions(self):
         self.create_action('quit', self.on_quit_action, ['<primary>q'])
         self.create_action('about', self.on_about_action)
+        self.create_action('toggle_sidebar', self.toggle_sidebar, None)
+        self.create_action('mode_none', self.set_mode, None, InteractionMode.NONE)
+        self.create_action('mode_pan', self.set_mode, None, InteractionMode.PAN)
+        self.create_action('mode_zoom', self.set_mode, None, InteractionMode.ZOOM)
+        self.create_action('mode_select', self.set_mode, None, InteractionMode.SELECT)
         self.create_action('preferences', preferences.open_preferences_window, ['<primary>p'], self)
```

Comment:
We should probably bind some shortcuts to some of these actions, but as said it's probably best to do that in a separate PR